source/manual/world-taxonomy.html.md

```diff
@@ -0,0 +1,71 @@
+In order to avoid having a large number of near identical taxons world taxons are based on an associated taxon structure. There are a small number of generic taxons that should be displayed on each of the individual world pages, for example [passports and emergency travel documents][6]. These are then mirrored in the world pages, for example [passports and emergency travel documents Vietnam][7], which has information specific to that country.
+
+[5]: https://github.com/alphagov/publishing-api/blob/master/lib/expansion_rules.rb#L29
+[6]: https://www.gov.uk/world/passports-and-emergency-travel-documents
+[7]: https://www.gov.uk/world/passports-and-emergency-travel-documents-vietnam
+
+## Tagging
+
+Unlike the rest of the taxonomy only content associated with [specific organisations][8] is allowed to be tagged to world taxons (DFID, FCO, UKVI, DIT, DExEU) and only [particular document types][9] (guidance, forms, detailed guides, and document collections) may be tagged to them. Presumably this is because the world pages are intended to be curation mechanisms for FCO and similar rather than content tagged to a topic.
+
+These rules are enforced by Whitehall but not by content-tagger meaning that these rules are not enforced uniformly.
+
+[8]: https://github.com/alphagov/whitehall/blob/56006c6f6ba033fbe450ef91d46204499e62e337/config/worldwide_tagging_organisations.yml
+[9]: https://github.com/alphagov/whitehall/blob/master/app/models/edition/taggable_organisations.rb#L4-L9
+
+## Concerns
+
+The separation of the world taxonomy has led it to become somewhat of a contentious issue. There have been a number of protracted discussions about whether this concept represents [technical debt for GOV.UK][10] tech community or if it is a form of product debt. Their existence is not well known among people who work on GOV.UK, despite it being a knowledge prerequisite to do any development work with the regular topic taxonomy.
+
+We also have a format of [world location][11] that was the predecessor to the world taxon pages. This is still used for a number of things ([atom feed][12], tagging in Whitehall, rendering the [world page][13]) so we’ve effectively got two concepts of the same thing and need to always keep them in sync.
+
+They are rendered in a completely different format to regular topic taxons. Compare https://www.gov.uk/education/running-and-managing-a-school with https://www.gov.uk/world/yemen.
+
+Content associated with a world location is tagged to the previous concept of a world location rather than a world taxon. For example https://www.gov.uk/government/news/information-and-events-for-uk-nationals-living-in-portugal has no data link to https://www.gov.uk/world/portugal and instead only has a link to the now defunct Portugal world location page. A [manual process][14] then creates a web link based on url guesswork.
+
```

Comment:
I've had a quick check and this appears to happen as for a document of `world_news_story` in whitehall, the UI doesn't show the user the world taxonomy and only allows tagging to the old stuff which is where this problem arises.

Reply:
Do you think this needs to be fixed?

Reply:
It feels like if we are wanting publishers to tag a `world_news_story` to a country we should expose the world taxonomy so they can do this via the taxonomy not just through the world location news page.

However this isn't really a concern of this documentation but would be good to jot down somewhere so some more thought could go into this as it wouldn't surprise me if other documents were unable to tag to the world taxonomy that probably should!

Reply:
yeah, fair play, am not going to take responsibility for finding it though, was definitely kev